src/corehost/cli/runtime_config.cpp

```diff
@@ -137,6 +158,15 @@ bool runtime_config_t::parse_opts(const json_value& opts)
         }
     }
 
+    if (rc)
+    {
+        if (m_has_roll_forward_option && m_has_roll_forward_on_no_candidate_fx_or_apply_patched_option)
+        {
+            trace::error(_X("It's invalid to use both `rollForward` and one of `rollForwardOnNoCandidateFx` or `applyPatches` in the same runtime config."));
```

Comment:
Should we throw here?

Reply:
We return false, which is used to indicate error (even exceptions from the JSON parser are eventually returned just as false).